inlong-manager/manager-dao/src/main/resources/generatorConfig.xml

```diff
@@ -114,7 +114,7 @@
                 enableCountByExample="false" enableDeleteByExample="false"
                 enableSelectByExample="false" enableUpdateByExample="false"/>
 
-        <table tableName="cluster_info" domainObjectName="ClusterInfoEntity"
+        <table tableName="third_party_cluster" domainObjectName="ClusterInfoEntity"
```

Comment:
The `domainObjectName` should change at the same time.

Reply:
I got it.